nginx/datadog_checks/nginx/data/conf.yaml.example

```diff
@@ -145,7 +145,7 @@ instances:
     ##     |__ aws_host
     ##     |__ aws_service
     ##
-    ## The `aws` auth type relies on boto3 to automatically gather AWS credentials, for example: from `.aws/credentials`.
+    ## The `aws` auth type relies on boto3 to automatically gather AWS credentials (e.g. from `.aws/credentials`).
```

Comment:
I'm surprised by the rest of these changes. Doesn't Nginx have a config spec?

Reply:
I am also unsure as to why these changes were made/are necessary.